<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newUser</class>
 <widget class="QDialog" name="newUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(&quot;:/images/background.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>701</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #FFF;

font-family: Inter;
font-size: 63px;
font-style: normal;
font-weight: 700;
line-height: normal;</string>
   </property>
   <property name="text">
    <string>Enter User Information</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_userName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>340</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
background: #1A7290;
color: #FFF;

text-align: center;
font-family: Inter;
font-size: 22px;
font-style: normal;
font-weight: 600;
line-height: normal;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Enter Unique Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>340</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
background: #1A7290;
color: #FFF;

text-align: center;
font-family: Inter;
font-size: 22px;
font-style: normal;
font-weight: 600;
line-height: normal;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Enter Email Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pass">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>270</y>
     <width>340</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
background: #1A7290;
color: #FFF;

text-align: center;
font-family: Inter;
font-size: 22px;
font-style: normal;
font-weight: 600;
line-height: normal;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Enter Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_age">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>380</y>
     <width>340</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
background: #1A7290;
color: #FFF;

text-align: center;
font-family: Inter;
font-size: 22px;
font-style: normal;
font-weight: 600;
line-height: normal;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Enter Age(18+)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Login">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>490</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 35px;
background: #D13526;
color: #FFF;

text-align: center;
font-family: Inter;
font-size: 35px;
font-style: normal;
font-weight: 600;
line-height: normal;
</string>
   </property>
   <property name="text">
    <string>Create!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
